admin_ui/app/permission/PermissionStore.js

```diff
@@ -0,0 +1,13 @@
+import { BehaviorSubject } from 'rxjs';
+
+class PermissionStore {
+  constructor() {
+    /*
+      The deleteModal$ observable takes 3 possible values:
+      "" (empty string), "DELETE", "DELETION_IN_PROGRESS"
```

Comment:
But do these values really show different modals, or is it the same modal with different arguments? Something to discuss in person. Also, see my comment regarding modal management above.

Reply:
We probably need to move this to use the same approach as you suggsted for the Org store: have the observable as `displayedModal$` and pass appropriate props to the deletion modal to show the "deletion in progress" animation (probably all modals should have an "IN_PROGRESS" prop....)